src/components/ScrollLock/tests/ScrollLock.test.tsx

```diff
@@ -29,23 +29,29 @@ describe('ScrollLock', () => {
 
     scrollLockContainer.instance().setState({showScrollLock: false});
 
-    expect(document.body.hasAttribute(`${SCROLL_LOCKING_ATTRIBUTE}`)).toBe(
-      true,
-    );
+    expect(
+      document.body.firstElementChild.hasAttribute(
+        `${SCROLL_LOCKING_WRAPPER_ATTRIBUTE}`,
```

Comment:
nitpick: There's no point to string interpolation if it's a single variable - same goes for other test cases below

```suggestion
        SCROLL_LOCKING_WRAPPER_ATTRIBUTE,
```

Reply:
Thank you for confirming my doubts, I was totally wondering why those were in there in the first place! Will fix as I rework the tests.